server/database/build.sql

```diff
@@ -0,0 +1,24 @@
+BEGIN;
+DROP TABLE IF EXISTS users, tickets_status, tickets CASCADE;
+CREATE TABLE users(
+    id SERIAL PRIMARY KEY,
+    email TEXT NOT NULL UNIQUE,
+    name VARCHAR NOT NULL ,
+    password VARCHAR NOT NULL
+);
+CREATE TABLE tickets_status(
+    uid SERIAL PRIMARY KEY,
+    status VARCHAR NOT NULL
+);
+CREATE TABLE tickets(
+    uid SERIAL PRIMARY KEY,
+    from_tickets VARCHAR NOT NULL,
+    to_tickets VARCHAR NOT NULL,
+    date INTEGER ,
```

Comment:
@alaabadra You could use `DATETIME` here, that might make the database easier to work with.

Reply:
ok, i chenged it

thankyou for this comment